Debounce/Bufferer.cs

```diff
@@ -0,0 +1,118 @@
+﻿// SPDX-FileCopyrightText: 2024 Frans van Dorsselaer
+//
+// SPDX-License-Identifier: MIT
+//
+// SPDX-FileContributor: Alain van den Berg
+
+namespace Dorssel.Utilities;
+
+/// <summary>
+/// A <see cref="Debouncer"/> that buffers events of type <typeparamref name="TEvent"/> that were triggered, aggregates them and sends aggregated events.
```

Comment:
Please make a note here that this one isn't as performant as the unbuffered Debouncer, due to allocations.

Reply:
Done